index.html

```diff
@@ -0,0 +1,292 @@
+          <span class="title-programe">Forum</span>
+          <p class="programe-para">
+            Listen to diffrerent ways of improving our service to have a better
+            experience.
+          </p>
+        </div>
+        <div class="programe-detail">
+          <img
+            src="./images/phone-view-icons/program_icon_04.png"
+            alt="conference-icon4"
+          />
+          <span class="title-programe">Exchange</span>
+          <p class="programe-para">
+            Opportunity open to recrute good personnel.
+          </p>
+        </div>
+        <div class="programe-detail">
+          <img
+            src="./images/phone-view-icons/program_icon_05.png"
+            alt="conference-icon5"
+          />
+          <span class="title-programe">Degustation</span>
+          <p class="programe-para">
+            party time and enjoyement will be available.
+          </p>
+        </div>
+      </div>
+      <div class="btn-mobile-program">SEE THE WHOLE PROGRAM</div>
+      <button type="button" class="btn-program">Join Us</button>
+    </section>
+    <section class="investors">
+      <h3>Featured Investors</h3>
+      <div class="orange-line"></div>
+      <div class="investors-container">
+        <div class="investors-list">
+          <img
+            src="./images/phone-view-icons/investors_01.png"
+            alt="investors-icon1"
+            class="investors-img"
+          />
+          <div class="investors-content">
+            <h4>Yochai Benkler</h4>
+            <h5>Chef in Hotel-hilton</h5>
+            <hr class="investors-line" />
+            <p class="investors-description">
+              Focusing on a common approach in how to improve the quality of the
+              service in a restaurant.
+            </p>
+          </div>
+        </div>
+        <div class="investors-list">
+          <img
+            src="./images/phone-view-icons/investors_02.png"
+            alt="investors-icon2"
+            class="investors-img"
+          />
+          <div class="investors-content">
+            <h4>Kilnam Chon</h4>
+            <h5>Chef in Hotel-palace</h5>
+            <hr class="investors-line" />
+            <p class="investors-description">
+              Discover achu and yellow soupe, explaine how it can cure some
+              certain illness.
+            </p>
+          </div>
+        </div>
+        <div class="investors-list">
+          <img
+            src="./images/phone-view-icons/investors_03.png"
+            alt="investors-icon3"
+            class="investors-img"
+          />
+          <div class="investors-content">
+            <h4>sohyeong Noh</h4>
+            <h5>Chef in Hotel-china-town</h5>
+            <hr class="investors-line" />
+            <p class="investors-description">
+              coming directly from HongKong to give her advice and encourage of
+              this new restaurant.
+            </p>
+          </div>
+        </div>
```

Comment:
- Kindly, Use `article` semantic tags for your investor list to keep semantic tags for better accessibility and search engines 👍